elfi/methods/methods.py

```diff
@@ -515,6 +544,16 @@ def _update_objective(self):
         self.objective['n_batches'] = ceil(n_batches)
         logger.debug('Estimated objective n_batches=%d' % self.objective['n_batches'])
 
+    def plot_state(self, **options):
+        displays = []
+        if options.get('interactive'):
+            from IPython import display
```

Comment:
You can call this method also directly if you loop the inference yourself, or just want the IPython plotted version so the import should be here.